CONTRIBUTING.md

```diff
@@ -66,3 +66,26 @@ GIT_USER=<Your GitHub username> USE_SSH=true yarn deploy
 ```
 
 If you are using GitHub pages for hosting, this command is a convenient way to build the website and push to the `gh-pages` branch.
+
+### Search
+
+To build index for search
+
+```text
+1. you would need to sign up an account on https://www.algolia.com/users/sign_in
+2. create a project for the purpose of creating index
+3. add an INDEX_NAME called "docusaurus" for instance on same website
+4. You add API_key and give rights for ACLs search, addObject, deleteObject, editSearch for the key
+5. SITE_URL is the deployed site, but siteindex needs to match it in order crawl it
+6. If u look at the pull request ff0a9a5 i have a sample .env file with the variables set.I also have a an example of what values needed as secret in scrape.sh
```

Comment:
Use proper spelling, punctuation, and formal language:

```
If you look at the pull request ff0a9a5, I have a sample .env file with the variables set. I also have an example of which secret values are needed in scrape.sh
```

Also, link to anything that you're talking about.